### PR TITLE
ADD: Bot behavior in the chat

### PR DESCRIPTION
**Pull request by tic-hack**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -99,6 +106,56 @@ class SimpleDialogueManager:
...
+        """
+        This function improves the bot conversation in the Telegram chat
+
+        :param question str: Command/question in the chat
+        """
+
+        random_proposal = ["Enjoy your day!",
+                           "Who is ready for a party?",
+                           "/newpoll, Do you like the word 'Badger'?",
+                           "I'm playing CS guys! join me",
```
Comment (daisukixci) [inline main.py]: LOOOL

File: main.py [hunk 2]
```diff
@@ -99,6 +106,56 @@ class SimpleDialogueManager:
...
+        # Emoji faces have to be encoded in unicode to be display in Telegram chat
+        self.emoji_dict = {":grinning:": "\U0001F600",
+                           ":joy:": "\U0001F602",
+                           ":rolling_laughing:": "\U0001F923",
+                           ":wink:": "\U0001F609",
+                           ":zany_face:": "\U0001F92A"}
+
+        self.article_opinion = ["So cool!", "Nice!",
+                                "Interesting! keep me in touch if you get more details on it",
+                                "Cool! but I've already read it"]
```
Comment (daisukixci) [inline main.py]: You miss friends, right ?

File: main.py (hunk 1 above)
Comment (tic-hack) [reply main.py]: I fix the line 134, when the bot uses /newpoll, the command doesn't work

Comment (tic-hack) [reply main.py]: If you have more ideas let me know :)

File: main.py (hunk 2 above)
Comment (tic-hack) [reply main.py]: What do you mean?

Comment (daisukixci) [reply main.py]: JK on the fact that you want the bot to react to post xD

Comment (tic-hack) [reply main.py]: Oh yeah! If no one answer you about a http link shared, the bot will answer, and so you'll be happy :D

**Comment by daisukixci** [comment]
Think to keep the README updated

File: main.py [hunk 3]
```diff
@@ -105,6 +114,58 @@ class SimpleDialogueManager:
...
+        This function improves the bot conversation in the Telegram chat
+
+        :param question str: Command/question in the chat
+        """
+
+        random_proposal = ["Enjoy your day!",
+                           "Who is ready for a party?",
+                           "I'm playing CS guys! join me",
+                           "How are you guys?",
+                           "The weather is really bad!"]
```
Comment (daisukixci) [inline main.py]: Why `random_proposal` is not managed like `article_opinion` and `emoji_dict`.

Maybe name `emoji_dict` emojis`, what do you think ?

Comment (tic-hack) [reply main.py]: Yeah! I agree with you. I modified both proposals

**Pull request merged**